### ISSUE TITLE
Field does not appear to work inside Vizy blocks.

### ISSUE THREAD

**Issue by tomkiss** [besteadfast/craft-preparse-field#92]
I've found the Preparse field to be incredibly helpful across a few projects now, thank you!

Perhaps the first limitation I've discovered is that the field doesn't appear to do anything when placed inside a [Vizy](https://github.com/verbb/vizy) block.

Due to Vizy's excellent editor experience, I've been switching over matrix article fields to [Vizy](https://verbb.io/craft-plugins/vizy/features) recently ([Vizy Github over here](https://github.com/verbb/vizy)). But oddly enough, when a Preparse field is used inside a Vizy block, it doesn't appear to do _anything_. There are no apparent errors, the field will load in the CP and act normal. However, nothing will render in the field. When inserted as a text input, even plain text placed inside the Preparse field does not render in the CP.

I appreciate it must be a minefield ensuring any reasonable degree of compatibility with all the different other plugins out there, but I thought I should report this nonetheless. 🙌

**Comment by tomkiss** [comment]
I did a bit of poking around in the source code and it seemed as if this may fall with the Vizy side of things, but I can't be sure as I'm not that confident with what I'm looking at. I've posted [over there](https://github.com/verbb/vizy/issues/195) just to be sure.

**Comment by jalendport** [comment]
Looks like this was resolved in Vizy [2.0.11](https://github.com/verbb/vizy/releases/tag/2.0.11) 👏

**Comment by tomkiss** [comment]
Yeah, I spent a bit of time looking at this yesterday. First, it didn't work, then it appeared to work when I added a field to the layout, and then it stopped working before later seemingly working again.
I ended up going around in circles trying to work out what was happening but ran out of time in the end. I don't think it's working properly but I also don't know which plugin needs attention in order to work properly.
